moveit_ros/moveit_servo/src/servo_calcs.cpp

```diff
@@ -294,17 +296,19 @@ void ServoCalcs::calculateSingleIteration()
   have_nonzero_twist_stamped_ = latest_nonzero_twist_stamped_;
   have_nonzero_joint_command_ = latest_nonzero_joint_cmd_;
 
+  planning_scene_monitor_->requestPlanningSceneState();
+  planning_scene_monitor::LockedPlanningSceneRO planning_scene(planning_scene_monitor_);
```

Comment:
This will lock the planning scene until this function exits, is this the desired thing .? you may want to wrap it in a new scope

Reply:
What Jafar means is something like this:

```
{
  planning_scene_monitor::LockedPlanningSceneRO planning_scene(planning_scene_monitor_);
  
  ... use planning_scene ...
}
```

Reply:
Done.

Reply:
I changed all calls to the locked planning scene into a function [here](https://github.com/ros-planning/moveit/pull/2894/commits/96175854ce790d86628d7cf1735183dccb82a3ff). Happy to get feedback if this is a good way to do it/whats the preferred way.